extensions/wikia/Email/i18n/Welcome.i18n.php

```diff
@@ -0,0 +1,33 @@
+<?php
+$messages = [];
+
+$messages['en'] = array(
+	'emailext-welcome-subject' => "Welcome to Wikia, $1!",
+	'emailext-welcome-summary' => "Congrats, you’re a Wikian! You are now part of a universe created entirely for fans and authored by millions of other fans, just like you. <br><br>So what’s next?",
+	'emailext-welcome-profile-header' => "Create your profile",
+	'emailext-welcome-profile-description' => "Here's your chance to build your identity on Wikia, connect with others, and discover more about what you love.",
+	'emailext-welcome-mobile-header' => "Take Wikia everywhere",
+	'emailext-welcome-mobile-description' => "Download your favorite Wikia fan apps for [$1 iOS] or [$2 Android] and never, ever stop obsessing about your fandoms.",
+	'emailext-welcome-basics-header' => "Learn the basics",
+	'emailext-welcome-basics-description' => "Need a quick tutorial on how to edit a page, start a discusion, or upload stuff? Watch these [$1 helpful videos].",
+	'emailext-welcome-fandom-header' => "Find your favorite fandom",
+	'emailext-welcome-fandom-description' => "Discover thousands of wikias across movies, TV, games, and more. Follow them so you never miss one moment.",
+	'emailext-welcome-footer-community' => "Want more information? Find additional advice, answers, and friendly faces at [http://community.wikia.com/ Wikia Community Central].",
+	'emailext-welcome-footer-thanks' => "Thanks for joining the fan universe! <br>The Wikia Community Team",
```

Comment:
Again, we should probably break this into a separate message. We actually have that "The Wikia Community Team" message already [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/i18n/EmailConfirmation.i18n.php#L16).